### DISCUSSION TITLE
將月球、火星地表當成氧氣農場，新設備將有毒土壤轉出氧氣

### DISCUSSION THREAD

**Discussion by 110409538** [1101-GS4719-C/Python-Community-GS4719-C#467] [科技新知]
> 歐洲太空總署最近投入資金，支持希臘 Thales Alenia Space 團隊投入開發另一種可檢測月球、火星土壤中危及人體的有毒活性氧設備，且還能將這些危險物質轉化成氧氣。

> 月球和火星沒有太多水資源且大氣稀薄，當表面粗糙土壤直接受強烈紫外線照射便容易產生活性氧，不僅破壞生物化石遺跡且具有極強腐蝕性，能損壞機器甚至灼傷未受太空服防護的皮膚與肺部組織，帶來永久性損害。

> 這項研究最終指向一種定期自月球/火星土壤中提取氧氣的大型反應器，團隊將之稱為「氧氣農業」，根據估計，將設備建置在 1.2 公頃土地上並接受太陽紫外線照射幾小時，就能提供 1 名太空人維生所需的氧氣量。這樣一來,人類離生活在火星的願景又更近了一步。

![image](https://user-images.githubusercontent.com/90551832/158007394-59a28a1f-056c-4e50-a28e-4cffa1c17576.png)
新聞來源:https://technews.tw/2022/03/11/farm-oxygen-moon-mars-esa/

**Comment by chupochun** [comment]
這樣看來人類距離移民火星的目標已經不遠了

**Comment by xkllkx** [comment]
馬斯克的火星計畫又前進了一步

**Comment by PoooYen** [comment]
1.2公頃才提供一個人，中央大學也才60公頃不就只能供應50個人XD

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望未來真的能去火星。

**Comment by cordydeng** [comment]
這個技術能夠降低我們的能源消耗，不僅對太空探索及環境都有很大的幫助

**Comment by HsuChen88** [comment]
搞不好火星可以直接像電影一樣直接種植物..?

**Comment by Sazmafuji** [comment]
計畫一步一步再邁進，可能明年就可以上火星

**Comment by HaoFish123** [comment]
感謝分享，宇宙的時代終於要來了嗎?

**Comment by treeann** [comment]
原來如此!感謝分享!

**Comment by stan890425** [comment]
火星真是一個可開發的行星

**Comment by jcpan821** [comment]
但這樣不就沒地方住了

**Comment by Ben6uo** [comment]
真的有機會哎 火星曾經有過河床 再多研究一下可能就可以開發了

**Comment by alexthebigpp** [comment]
太棒了，希望可以趕快把人送上去。

**Comment by etin2468** [comment]
1.2公頃一個人，好孤單...

**Comment by Lynn12378** [comment]
除了氧氣，有沒有液態的水資源可以供人類生存也是很難克服的一大難題。

**Comment by Jimmy911218** [comment]
感覺離出去外星的時間更近了

**Comment by yenwei12138** [comment]
可以採集氧氣很不錯，但人類能不能安全的用那些氧氣又是另一問題

**Comment by Hchejxnc** [comment]
感覺距離目標還有很長遠的路要走~

**Comment by harrylee1971** [comment]
我只想在地球好好生活，火星好可怕，一個不小心我人就死翹翹了
